arrays/arrays.c

```diff
@@ -116,13 +148,25 @@ void arr_append(Array *arr, char *element) {
  *****/
 void arr_remove(Array *arr, char *element) {
 
+  int index = 0;
+
   // Search for the first occurence of the element and remove it.
   // Don't forget to free its memory!
-
-  // Shift over every element after the removed element to the left one position
-
-  // Decrement count by 1
-
+  for (int i = 0; i < arr->count; i++) {
+    if (arr->elements[i] == element) {
```

Comment:
You should use `strcmp(string1, string2) == 0` in stead of `string1 == string2`. I think the code will still work without it, but it will draw a warning and it's not best practice.